src/mgr/Action.cc

```diff
@@ -112,11 +113,18 @@ Mgr::Action::fillEntry(StoreEntry* entry, bool writeHttpHeader)
         entry->replaceHttpReply(rep);
     }
 
-    dump(entry);
+    dump(entry); // TODO: replace with print() when all children are converted
 
     entry->flush();
 
     if (atomic())
         entry->complete();
 }
 
+void
+Mgr::Action::dump(StoreEntry *entry)
+{
+    PackableStream out(*entry);
+    print(out);
```

Comment:
Sure, okay. Will do from here on.

Reply:
Thank you. BTW, for YAML-compliant output, we should use a more specific name `yaml` instead of `os` IMO. However, that (future) naming idea does not affect this PR.